r/R/arrow-datum.R

```diff
@@ -103,10 +103,10 @@ Ops.ArrowDatum <- function(e1, e2) {
 #' @export
 Math.ArrowDatum <- function(x, ..., base = exp(1), digits = 0) {
   switch(.Generic,
-    abs = ,
-    sign = ,
-    floor = ,
-    ceiling = ,
+    abs = eval_array_expression("abs_checked", x),
+    sign = eval_array_expression("sign", x),
+    floor = eval_array_expression("floor", x),
+    ceiling = eval_array_expression("ceil", x),
```

Comment:
For example, for `abs`, I think the `Math.ArrowDatum` fails because it cannot find it, since it's been renamed to `base::abs()` in the `.unary_function_map`. This ☝🏻 was my work around to make sure `abs` is mapped to `abs_checked`.

Reply:
Done.